pkg/service/nodes.go

```diff
@@ -192,3 +203,7 @@ func drainNodeUtil(node *v1.Node, DrainTimeout int, client kubernetes.Interface)
 	}
 	return err
 }
+
+func deleteNodeUtil(node *v1.Node, client kubernetes.Interface) error {
+	return nil
```

Comment:
so we are not actually deleting the node?

Reply:
Lol. Wrong Branch again.